PerformanceMetrics/PerformanceMetrics.h

```diff
@@ -75,6 +75,14 @@ namespace Plugin {
 
                 _parent.PluginDeactivated(*service);
             }
+            void Activation (const string&, PluginHost::IShell* service) override
```

Comment:
As discussed on slack I now understand why these lines are needed, it is because of this patch file in the RDKServices build: https://gerrit.teamccp.com/plugins/gitiles/rdk/yocto_oe/layers/meta-rdk-video/+/23Q3_sprint/recipes-extended/wpe-framework/wpeframework/r4/activationevent.patch

However we cannot just merge this lines at that will break build(systems) that do not include the patch.
I will have a discussion on the need for this patch (if merging would be really necessary in the meantime may I propose to just as a workaround for now remove the override keyword and also not make them virtual, then it will build with and without the patch (and as downside only add a very small piece of unneeded code that perhaps the optimizer throws away anyway)

Reply:
Removed the new override apis as discussed.